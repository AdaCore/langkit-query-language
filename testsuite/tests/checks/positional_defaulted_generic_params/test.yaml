driver: 'checker'
rule_name: Positional_Actuals_For_Defaulted_Generic_Parameters
input_sources: ['pos.ads']
auto_fix: True
