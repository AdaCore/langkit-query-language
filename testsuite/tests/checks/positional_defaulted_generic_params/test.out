pos.ads:23:36: error: use named notation when passing actual to defaulted generic parameter
23 |                                    2,            --  FLAG
   |                                    ^ 

pos.ads:24:36: error: use named notation when passing actual to defaulted generic parameter
24 |                                    Fun_2,        --  FLAG
   |                                    ^^^^^ 

pos.ads:25:36: error: use named notation when passing actual to defaulted generic parameter
25 |                                    Fun_1);       --  FLAG
   |                                    ^^^^^ 

